static/app/types/profiling.d.ts

```diff
@@ -37,6 +37,7 @@ declare namespace Profiling {
     stack_id: number;
     thread_id: string;
     timestamp: number;
+    queue_address?: string;
```

Comment:
Continuous profiling doesnt use queue addresses

Reply:
Removed all of this